<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>682</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(41, 64, 123);
	color: rgb(97, 141, 250);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="back_btn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	qproperty-icon: url(:/icons/icons/arrow-left-solid.svg);
}</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>Загрузить данные с сайта</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="timeGrid">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>C:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDateEdit" name="dateTo">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
	border: none;
	spacing: 5px;
	

}

QDateEdit::drop-down{
	image: url(:/calendar/icons/calendar-regular.svg);
	
	width: 20px;
	height: 20px;
	subcontrol-position: right center;
	spacing: 5px;




	

}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimumDate">
        <date>
         <year>2000</year>
         <month>9</month>
         <day>14</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2022</year>
         <month>6</month>
         <day>14</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateFrom">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
	border: none;
	spacing: 5px;
	

}

QDateEdit::drop-down{
	image: url(:/calendar/icons/calendar-regular.svg);
	cursor: pointer;
	width: 20px;
	height: 20px;
	subcontrol-position: right center;
	spacing: 5px;




	

}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimumDate">
        <date>
         <year>2000</year>
         <month>9</month>
         <day>14</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2022</year>
         <month>6</month>
         <day>14</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>До:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>Временной интервал</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="cloudGrid">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>От:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cloudFrom">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>До:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="cloudTo">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>90</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>Облачность %:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="zonesGrid">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>Зоны интереса</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="openMap_btn">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-width: 2px;
	border-radius: 12px;
	border-color: #F5F5F5;
}

QPushButton:hover{
	
	color: #618DFA;
	border-color: #618DFA;
}</string>
       </property>
       <property name="text">
        <string>Отобразить карту</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="zonesList">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-width: 3px;
	border-radius: 10px;
	border-color: #618DFA;
}</string>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="zonesInfoList">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-width: 3px;
	border-radius: 10px;
	border-color: #618DFA;
}</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="pathGrid">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>Каталог загрузки: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dwnPath_label">
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #F5F5F5;
}</string>
       </property>
       <property name="text">
        <string>[Путь по умолчанию]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-bottom-width: 2px;
	
	border-color: #F5F5F5;
}

QPushButton:hover{
	
	color: #618DFA;
	border-color: #618DFA;
}</string>
       </property>
       <property name="text">
        <string>Сменить каталог</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="download_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-width: 2px;
	border-radius: 12px;
	border-color: #F5F5F5;

}

QPushButton:hover{
	
	color: #618DFA;
	border-color: #618DFA;
}</string>
         </property>
         <property name="text">
          <string>Скачать сцены</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="copy_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-width: 2px;
	border-radius: 12px;
	border-color: #F5F5F5;

}

QPushButton:hover{
	
	color: #618DFA;
	border-color: #618DFA;
}</string>
       </property>
       <property name="text">
        <string>Перенести</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QGridLayout" name="progressGrid">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="info_label">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(255, 6, 60)

}</string>
       </property>
       <property name="text">
        <string>Подробности</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RF.qrc"/>
 </resources>
 <connections/>
</ui>
